packages/kbn-check-mappings-update-cli/current_mappings.json

```diff
@@ -548,6 +548,15 @@
       },
       "actionTypeId": {
         "type": "keyword"
+      },
```

Comment:
I made a change in [`x-pack/plugins/actions/server/saved_objects/mappings.ts`](https://github.com/elastic/kibana/pull/162653/files#diff-f253cdd2684d8de6aedb8a571bd60cf55cb08fbf178c31488598247edbd89318R24), so I believe i will need it here too?

Reply:
We'd really prefer the mappings not be extended.  Can you not access this field from the script in `action_telemetry.ts`?  If for some reason it's not in `doc`, it should be in `source`.  I suspect you could also "add" a field during the telemetry query as a runtime-field added to the search.